### PR TITLE
Fix ES queries aggregation max size

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Choosing a query grouping size to 'no-limit' sets it to 500 in the generated Elasticsearch query. This is a strong limitation for our metrics as some unique counters (such as views) may be higher than that on large analysis periods.

## Proposal

To mitigate this issue, we give us some space by settings it to the maximal allowed value for our Elasticsearch cluster, e.g. 10000.

### CODE REVIEW

**Review by quitterie-lcs — APPROVED** [review]

**Comment by jmaupetit** [comment]
Looking for a smarter solution.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Unreleased]
 
+## [0.5.1] - 2022-05-03
+
+## Changed
```
Comment (jmaupetit) [inline CHANGELOG.md:13]: Should be `fixed` as it's a bug!

**Review by jmaupetit — COMMENTED** [review]
Almost good to go, thanks!

**Comment by quitterie-lcs** [comment]
@jmaupetit fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Unreleased]
 
+## [0.5.1] - 2022-05-03
```
Comment (jmaupetit) [inline CHANGELOG.md:11]:
```suggestion
## [0.5.1] - 2022-05-04
```

**Review by jmaupetit — COMMENTED** [review]
One small change and we are good to go (as I opened this PR, I cannot validate it).

**Pull request merged**